src/OhmmsApp/RandomNumberControl.cpp

```diff
@@ -117,7 +117,7 @@ void RandomNumberControl::make_children()
     o << "  Random seeds Node = " << rank << ":";
```

Comment:
First nprocs<4

Reply:
This one seems okay to remove.  The app_log output is only printed for rank 0 anyway.

Reply:
I don't mean remove just the if line but the whole if section.